File: sbt-test/android-sdk-plugin/hello-world-gradle-layout/src/main/java/com/example/app/MainActivity.java
package com.example.app;

import android.app.Activity;
import android.os.Bundle;

public class MainActivity extends Activity {
    @Override
    public void onCreate(Bundle b) {
        super.onCreate(b);
        setContentView(R.layout.hello);
        if (BuildConfig.DEBUG) {

        }
    }
}

File: sbt-test/android-sdk-plugin/proguard-cache-duplicate-classes/src/main/scala/com/example/proguardcache/MainActivity.scala
package com.example.proguardcache

import android.app.Activity
import android.os.Bundle

class MainActivity extends Activity with TypedFindView {
    override def onCreate(b: Bundle) {
        super.onCreate(b)
        setContentView(R.layout.main)
        findView(TR.text).setText("Hello again, world!")
        findView(TR.text)
    }
}
